### ISSUE TITLE
Re-Spawned Fluffies Age, but Don't Mature

### ISSUE THREAD

**Issue by Shadow8472** [eidalac/FluffyBreeder#30]
**Describe the bug**
Fluffy 3 has too weak of stats for my breeding program, so I sold him off. Fluffy 3 re-spawned when I went looking for ferals. This has happened enough times that I've focused on training her when she's in the store. I find it funny that she has the Swift trait, explaining how my little escape artist keeps turning up asking for "nyu housie."

The bug is that --despite her age displaying correctly-- she's still a foal at 1 year, 5 weeks, and 6 days old. 

**To Reproduce**
1. Sell young fluffies
2. Recognize a young fluffy you previously sold while hunting ferals
3. Take it back
4. Repeat 1-3 for a while
5. Observe familiar fluffy hasn't changed a bit since you sold him/her, including maturity

**Expected behavior**
Maturity should be based on age.

**Screenshots**
![image](https://user-images.githubusercontent.com/53035941/222953554-38abd470-56f7-4553-8535-6dbaa3d09ac1.png)

**Desktop (please complete the following information):**
 - OS: Manjaro Linux
 - Browser Firefox
 - Version 110.0 (64-bit)

**Additional context**
Fluffy Breeder version: 1.41.384-beta

**Comment by eidalac** [comment]
Yeah, the feral returning has a number of bugs, since the game only updates 'store' fluffies on normal ticks.
I thought I had corrected feral age but that might just have been on my to-do list.

**Comment by Shadow8472** [comment]
Also note that I'm not on the latest development branch.

Still, I found this frequently returning foal hilarious either way.
